Send message when spellbot does not have permission to manage reactions (#143)
lexicalunit/spellbot: CHANGELOG.md

- Adds meta theme-color and meta og:site_name to front page.
- Adds a note about Administrator permissions to the Commands for Admins section of the front page.

> - Spellbot will message in the channel when they do not have permission to adjust reactions

Reviewer: ```suggestion
- Spellbot will message in the channel when it does not have permission to adjust reactions.
```
